### DISCUSSION TITLE
How to extend core/link

### DISCUSSION THREAD

**Discussion by gabripuglia** [WordPress/gutenberg#54694] [Q&A]
Hi, i need to extend core/link used in RichText component with title input field and a toggle button  for "no follow". How can i reach this goal?

**Comment by t-hamano** [comment]
Hi @gabripuglia,

The `nofollow` attribute control for the inline link was implemented in #53945. Does this help you?

**Reply by gabripuglia** [reply]
Hi, I need to add title into core/link like input field

**Reply by t-hamano** [reply]
I don't think core/link can be expanded currently, but I've seen several opinions that require it. #11599 is one example.
